indico/modules/events/papers/blueprint.py

```diff
@@ -34,11 +34,15 @@
                  templates.RHDownloadPaperTemplate)
 _bp.add_url_rule('/contributions/<int:contrib_id>/paper/submit', 'submit_revision',
                  display.RHSubmitPaper, methods=('GET', 'POST'))
-# Paper reviews
+# Paper reviews/judgments
 _bp.add_url_rule('/papers/<int:contrib_id>/review/type/<review_type>', 'submit_review',
                  display.RHSubmitPaperReview, methods=('GET', 'POST'))
 _bp.add_url_rule('/papers/<int:contrib_id>/revision/<int:revision_id>/review/<int:review_id>/edit', 'edit_review',
                  display.RHEditPaperReview, methods=('GET', 'POST'))
+_bp.add_url_rule('papers/<int:contrib_id>/judge/', 'judge_paper',
```

Comment:
I was talking about the leading slash...

However, for judge/reset I would use the same style as we used in abstract: .../judge and .../reset (or .../reset-judgment) - no strong opinion though.